README.md

```diff
@@ -1,2 +1,181 @@
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://ezraermy.github.io/youcode-web-dev-courses/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need to:
+- Install Git
+- Install NPM
+- Have working browser
+- Good internet connection
+- Install a code editor like Visual Studio Code
+- Install Live Server plugging in Visual Studio Code
+
+### Setup
+
+Clone this repository to your desired folder:
+
+- Choose a directory
+- Open a Terminal
+- write: git clone https://github.com/ezraermy/ezra_portfolio/edit/portfolio_mobile_projects
+- get into to directory "portfolio_mobile_mainpage"
+- write: npm install
+
+### Install
+
+Install this project with linters:
+
+- On your branch create .github/workflows/linters.yml
+- Do not make any changes in config files
+- Create .gitignore/ and instert node_modules/
+- On the terminal run npm install --save-dev hint@7.x
+- Copy .hintrc to the root directory of your project.
+- Run npx hint .
+- Run npx eslint .
+- Fix linter errors.
+
+### Usage
+
+To run the project, execute the following command:
+
+- Open the project by launching the Live Server plugging in visual studio code
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Ezra Yehuwalashet**
+
+- GitHub: [@ezraermy](https://github.com/)
+- Twitter: [@ezraermy](https://twitter.com/)
+- LinkedIn: [Ezra Yehuwalashet](https://linkedin.com/)
+
+👤 **Cindy Shin**
+- Original Design: [by Cindy Shin](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)
+- Dribbble: [Cindy Shin](https://dribbble.com/adagio07/collections)
+- LinkedIn: [Cindy Shin](https://www.linkedin.com/in/adagio07)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Main_Page]**
+- [ ] **[Table_of_Content]**
+- [ ] **[About_Page]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Jose Luis Vazquez Flores
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
```

Comment:
- [ ] It would be better to give Cindy Shin the appropriate credit, by mentioning her here in the acknowledgment as per the project requirements :recycle: